mungegithub/mungers/publish_scripts/apimachinery_sync_from_kubernetes.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
```

Comment:
> We can get the script from the repo, but I think it's better to centralize all the publishing logic in test-infra. If a developer is going to change the publishing logic, probably he'll need to update the scripts and other parts of the robot at the same time. The workflow is much easier if all the pieces are in the same repo.

Thing is, no one but a googler can actually check on this script.  That really limits the pool of people who can contribute.

We're slipping pretty far out of date (need a sync pretty badly) so I wouldn't block on it, but I think this will make it harder to make further improvements.